### ISSUE TITLE
the data is not available

### ISSUE THREAD

**Issue by yangyijane** [vikjam/mostly-harmless-replication#8]
Thank you so much for the code. When I try to replicate, the data are not accessible. Could you help to sort it out

**Comment by yangyijane** [comment]
In download.file("http://economics.mit.edu/~dautor/outsourcingatwill_table7.zip",  :
  cannot open URL 'https://economics.mit.edu/~dautor/outsourcingatwill_table7.zip': HTTP status was '404 Not Found' 


The error occurred when I tried to replicate your work on DID chapter

**Comment by yangyijane** [comment]
And I also tried to pull out other data, but they are not available. Could you pls advise what the problem: 
In unzip("asciiqob.zip") : error 1 in extracting from zip file
> download.file('http://economics.mit.edu/files/397', 'asciiqob.zip')
trying URL 'http://economics.mit.edu/files/397'
Error in download.file("http://economics.mit.edu/files/397", "asciiqob.zip") : 
  cannot open URL 'http://economics.mit.edu/files/397'
In addition: Warning messages:
1: In download.file("http://economics.mit.edu/files/397", "asciiqob.zip") :
  downloaded length 0 != reported length 22567
2: In download.file("http://economics.mit.edu/files/397", "asciiqob.zip") :
  cannot open URL 'https://economics.mit.edu/files/397': HTTP status was '404 Not Found'

**Comment by 5685918** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            
        
    
    oh, I'm so sorry，I'm also forwarding someone else's code.
    
     
    
        
                    ***@***.***
        
    
 

  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            Yi ***@***.***>
            
        
      
      
        
         Date 
        
        
        9/30/2022 10:02
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [vikjam/mostly-harmless-replication] the data is not available (Issue #8)
        
      
    
  
  
In download.file("http://economics.mit.edu/~dautor/outsourcingatwill_table7.zip",  :
cannot open URL 'https://economics.mit.edu/~dautor/outsourcingatwill_table7.zip': HTTP status was '404 Not Found'
The error occurred when I tried to replicate your work on DID chapter

**Comment by yangyijane** [comment]
thank you, is it r code to open those data? Could you elaborate it a bit more? I still do not know how to do it.

**Comment by yangyijane** [comment]
> font{ line-height: 1.6; } ul,ol{ padding-left: 20px; list-style-position: inside; } oh, I'm so sorry，I'm also forwarding someone else's code. ***@***.***
> […](#)
> ---- Replied Message ---- From Yi ***@***.***> Date 9/30/2022 10:02 To ***@***.***> Cc ***@***.***> Subject Re: [vikjam/mostly-harmless-replication] the data is not available (Issue #8) In download.file("http://economics.mit.edu/~dautor/outsourcingatwill_table7.zip", : cannot open URL 'https://economics.mit.edu/~dautor/outsourcingatwill_table7.zip': HTTP status was '404 Not Found' The error occurred when I tried to replicate your work on DID chapter —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

thank you. Could you pls explain it a bit? I don't know how to use this code.

**Comment by vikjam** [comment]
Thanks for submitting the issue @yangyijane. Seems like the file has moved. If you change the URL it should be able to download the file. I'll update the code when I have a chance.

```r
# Download the data and unzip it
download.file(
  "https://www.dropbox.com/s/m6o0704ohzwep4s/outsourcingatwill_table7.zip?dl=1",
  "outsourcingatwill_table7.zip"
)
unzip("outsourcingatwill_table7.zip")
```

**Comment by 5685918** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            You have a wechat? Or other contact information, which may be more convenient for communication. Because I don't often check my email.
        
    
    
        
            
        
    
    
     
    
        
                    ***@***.***
        
    
 

  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            ***@***.***>
            
        
      
      
        
         Date 
        
        
        10/2/2022 11:17
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [vikjam/mostly-harmless-replication] the data is not available (Issue #8)
        
      
    
  
  
Thanks for submitting the issue @yangyijane. Seems like the file has moved. If you change the URL it should be able to download the file. I'll update the code when I have a chance.
# Download the data and unzip it
download.file(
  "https://www.dropbox.com/s/m6o0704ohzwep4s/outsourcingatwill_table7.zip?dl=1",
  "outsourcingatwill_table7.zip"
)
unzip("outsourcingatwill_table7.zip")

**Comment by vikjam** [comment]
Fixed in commit https://github.com/vikjam/mostly-harmless-replication/commit/b669edc1401a739fd01666062bdb0d44c698822b

**Comment by 5685918** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        
            
        
    
    
        
            I don't know what this is, and I can't get in.
    
     
    
        
                    ***@***.***
        
    
 

  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            ***@***.***>
            
        
      
      
        
         Date 
        
        
        10/2/2022 12:12
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [vikjam/mostly-harmless-replication] the data is not available (Issue #8)
        
      
    
  
  
Closed #8 as completed.

**Comment by yangyijane** [comment]
thank you
